doc/source/ray-air/doc_code/checkpoint_usage.py

```diff
@@ -23,18 +23,12 @@ def build_model() -> tf.keras.Model:
 
 
 # __use_trainer_checkpoint_start__
-import pandas as pd
 import ray
 from ray.air import train_test_split
 from ray.train.xgboost import XGBoostTrainer
 
 
-bc_df = pd.read_csv(
-    "https://air-example-data.s3.us-east-2.amazonaws.com/breast_cancer.csv"
-)
-dataset = ray.data.from_pandas(bc_df)
-# Optionally, read directly from s3
-# dataset = ray.data.read_csv("s3://air-example-data/breast_cancer.csv")
+dataset = ray.data.read_csv("s3://air-example-data/breast_cancer.csv")
```

Comment:
This still needs fixing -- the others seem to be working now :)

Reply:
addressed! and more!